### PR TITLE
add support for ceph-volume inventory command

### PR DESCRIPTION
**Pull request by noahdesu**

### CODE REVIEW

File: infrastructure-playbooks/storage-inventory.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  hosts:
+    - "{{ osd_group_name|default('osds') }}"
+
+  become: true
+
+  tasks:
+    - import_role:
+        name: ceph-defaults
+
+  post_tasks:
```
Comment (leseb) [inline infrastructure-playbooks/storage-inventory.yml]: the `post_tasks` is not really needed here, you can simply put this task as part of `tasks` above.

File: tox.ini [hunk 2]
```diff
@@ -283,6 +292,8 @@ changedir=
   rgw_multisite: {toxinidir}/tests/functional/centos/7/rgw-multisite
   rgw_multisite_container: {toxinidir}/tests/functional/centos/7/rgw-multisite-container
   container_podman: {toxinidir}/tests/functional/fedora/29/container-podman
+  storage_inventory: {toxinidir}/tests/functional/centos/7/cluster
```
Comment (leseb) [inline tox.ini]: I'd run a smaller scenario instead of the entire roles, for example: `lvm_osds` and `lvm_osds_container`.

**Comment by noahdesu** [comment]
donzo

**Comment by noahdesu** [comment]
jenkins test dev-container_podman

**Comment by noahdesu** [comment]
jenkins retest

**Comment by noahdesu** [comment]
@leseb fixed the errors in travis, and the dev-container_podman tests passed. the failure in the jenkins pipeline seems completely unrelated. do you see anything there i might have missed?

**Pull request merged**